Toggl.Core.UI/ViewModels/Settings/Siri/SiriShortcutsViewModel.cs

```
@@ -42,6 +42,6 @@ public sealed class SiriShortcutsViewModel : ViewModel
             navigationService.Navigate<SiriShortcutsCustomTimeEntryViewModel>();
 
         public IObservable<IThreadSafeProject> GetProject(long projectId)
-            => interactorFactory.GetProjectById(projectId).Execute();
+            => interactorFactory.GetProjectById(projectId).Execute().Catch(Observable.Empty<IThreadSafeProject>());
```

I'm maybe missing some more context, but isn't it bad to just swallow all the errors and act in that case as if there are no projects with that id?